WeatherApp/WeatherApp/Feature/CityScreen/CityScreenView.swift

```diff
@@ -0,0 +1,86 @@
+import SwiftUI
+
+struct CityScreenView: View {
+
+    @ObservedObject var viewModel: CityScreenViewModel
+
+    let columns = [
```

Comment:
private

Reply:
Only columns can be made private, 'CityScreenView' initializer is inaccessible due to 'private' protection level